taskfiles/k8s.yml

```diff
@@ -23,9 +23,19 @@ tasks:
           paths='./...' \
           crd \
           webhook \
-          rbac:roleName=manager-role \
-          output:crd:artifacts:config=config/crd/bases \
-          output:rbac:artifacts:config=config/rbac/bases/operator
+          output:crd:artifacts:config=config/crd/bases
+      - |
```

Comment:
nit: I think naming these or prefixing them `v1` and `v2` would clarify the intent here?

Reply:
Prefixing you mean comment?

Reply:
Instead of `output:rbac:artifacts:config=config/rbac/redpanda-role` `output:rbac:artifacts:config=config/rbac/v2-redpanda-role`. The words `cluster` and `redpanda` are so overloaded they don't really signify anything on their own if you don't already know what's being discussed.